requirements.txt

```diff
@@ -0,0 +1,7 @@
+cytoolz>=0.9.0,<1.0.0
+eth-utils>=1.0.1,<2.0.0
+rlp>=0.4.7,<1.0.0
+eth-tester==0.1.0b21
```

Comment:
As Piper pointed out on gitter, `eth-tester` only need to be included 
 `dev_requirement.py`.